website/docs/r/one_dashboard.html.markdown

```diff
@@ -169,6 +241,12 @@ All nested `widget` blocks support the following common arguments:
   * `width` - (Optional) Width of the widget.  Valid values are `1` to `12` inclusive.  Defaults to `4`.
   * `height` - (Optional) Height of the widget.  Valid values are `1` to `12` inclusive.  Defaults to `3`.
   * `ignore_time_range` - (Optional) With this turned on, the time range in this query will override the time picker on dashboards and other pages. Defaults to `false`.
+  * `facet_show_other_series` - (Optional) With this turned on, this customization allows you to have the Other groups option visible or not if a facets ona query returns are more than 2000 for bar charts, pie charts, tables, the Other groups facet aggregates the rest of the facets. Defaults to `false`
+  * `y_axis_left_min`, `y_axis_left_max` - (Optional) Adjust the Y axis to display the data within certain values by setting a minimum and maximum value for the axis for line charts and area charts. If no customization option is selected, dashboards automatically displays the full Y axis from 0 to the top value plus a margin.
+  * `legend_enabled` - (Optional) wWith this turned on, the legend will be displayed. Defaults to `false`.
```

Comment:
Typo `wWith` and is this correct? Don't we show the legend by default? So the default would be true?

Reply:
corrected typo

Reply:
The default for `legend_enabled` is true, not false. Just double checked in UI.